src/index.tsx

```diff
@@ -288,13 +294,18 @@ export class App extends React.Component<any, AppState> {
       elements = clearSelection(elements);
       this.forceUpdate();
       this.setState({ elementType: "selection" });
+      event.preventDefault();
       if (window.document.activeElement instanceof HTMLElement) {
         window.document.activeElement.blur();
       }
-      event.preventDefault();
       return;
     }
-    if (isInputLike(event.target)) return;
+
+    if (isToolIcon(event.target)) event.target.focus();
```

Comment:
This won't do anything since the `event.target` will either be the old button, or `body`

Reply:
So you'll prob need to use `document.querySelector` to get the target elem